c7n/resources/awslambda.py

```diff
@@ -799,3 +799,52 @@ def format_resource(self, r):
         payload.update(self.filter_empty(
             select_keys(r, ['Version', 'CreatedDate', 'CompatibleRuntimes'])))
         return envelope
+
+
+@AWSLambda.filter_registry.register('lambda-edge')
+
+class LambdaEdgeFilter(Filter):
+    """
+    Filter for lambda@edge functions
+
+    :example:
+
+        .. code-block:: yaml
+
+            policies:
+                - name: lambda-edge-filter
+                  resource: lambda
```

Comment:
its worth documenting, and adding to the example (re region condition) and perhaps worth short circuiting the filter in the state: true case if region isn't aligned, as afaics lambda edge only exists in us-east-1 afaict

Reply:
Thanks for the feedback @kapilt. Yup, lambda@edge only exists in us-east-1. I have gone ahead and addressed your feedback